docs/semgrep-secrets/generic-secrets.md

```diff
@@ -0,0 +1,60 @@
+---
+slug: generic-secrets
+title: Scan for generic secrets
+hide_title: true
+description: Use Semgrep to identify generic secrets in your code.
+tags:
+ - Semgrep Assistant
+ - Semgrep Secrets
+---
+
+import PL from '@site/src/components/Placeholder';
+
+# Generic secrets (beta)
+
+:::info
+Generic secrets is in beta. Please contact [Support](/support) to request access to this feature.
```

Comment:
```suggestion
Generic secrets is in public beta. Please contact [Support](/support) to request access to this feature.
```
I assume it's in public beta (i.e. anyone can opt-in) based on the fact that the toggle is available from Settings. Is that correct?

Reply:
I checked w/ Milan, and it is in public beta